tests/test_theclevercarrot_2.py

```diff
@@ -0,0 +1,97 @@
+from recipe_scrapers._grouping_utils import IngredientGroup
+from recipe_scrapers.theclevercarrot import TheCleverCarrot
+from tests import ScraperTest
+
+
+class TestTheCleverCarrotScraper(ScraperTest):
+
+    scraper_class = TheCleverCarrot
+    test_file_name = "theclevercarrot_2"
+
+    def test_host(self):
+        self.assertEqual("theclevercarrot.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("The Clever Carrot", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual(
+            "dinner tonight: sweet chili shrimp stir-fry", self.harvester_class.title()
+        )
+
+    def test_category(self):
+        self.assertEqual(None, self.harvester_class.category())
```

Comment:
Nitpick: no need to add test coverage for the `category` field if it isn't implemented by the scraper.